ci-operator/config/openshift/boilerplate/openshift-boilerplate-master.yaml

```diff
@@ -0,0 +1,21 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
```

Comment:
I'm curious if this means we end up pulling a bunch of stuff we don't need into the container (there's no golang in the boilerplate repo, nor will there be for the forseeable future). But I don't know if removing this line will break the world, so meh.

Reply:
Yeah, this image contains various tools like make, etc.

Reply:
That's cool; I'm saying we could surely live with a much lighter image, because we don't need golang at all.

But still meh.